fleet/service.go

```diff
@@ -92,15 +95,19 @@ func (svc fleetService) thing(token, id string, name string, md map[string]inter
 	return thing, nil
 }
 
-func NewFleetService(logger *zap.Logger, auth mainflux.AuthServiceClient, agentRepo AgentRepository, agentGroupRepository AgentGroupRepository, agentComms AgentCommsService, mfsdk mfsdk.SDK) Service {
+func NewFleetService(logger *zap.Logger, auth mainflux.AuthServiceClient, agentRepo AgentRepository, agentGroupRepository AgentGroupRepository, agentComms AgentCommsService, mfsdk mfsdk.SDK, aTicker *time.Ticker, aDone chan bool) Service {
 
-	return &fleetService{
+	service := fleetService{
```

Comment:
I think we can create aTicket and aDone here, rather than accepting them as arguments